bin/cloneDataDump.js

```diff
@@ -0,0 +1,46 @@
+const clone = require('git-clone');
+const rimraf = require('rimraf');
+const fs = require('fs');
+const path = require('path');
+
+const tmpDestination = path.resolve(`.${path.sep}tmp${path.sep}dataDump${path.sep}`);
+const tmpData = path.resolve(`${tmpDestination}${path.sep}data${path.sep}`);
+const employersDestination = path.resolve(`.${path.sep}src${path.sep}data${path.sep}employers${path.sep}`);
+
+const cleanupTmpDirectory = () => rimraf(tmpDestination, () => {
+  // eslint-disable-next-line no-console
+  console.log(`${tmpDestination} has been removed.\n`);
+});
+
+const transferEmployerFiles = () => {
+  fs.readdirSync(tmpData).forEach((file) => {
+    if (path.extname(file) === '.json') {
+      fs.renameSync(`${tmpData}${path.sep}${file}`, `${employersDestination}${path.sep}${file}`, (err) => {
+        if (err) throw err;
+      });
+    }
+  });
```

Comment:
consider using a filter/map pattern here instead of nesting if statements
```suggestion
   fs.readdirSync(tmpData)
    .filter((file) => file.extname !== '.json')
    .map((file) => {
      fs.renameSync(`${tmpData}${path.sep}${file}`, `${employersDestination}${path.sep}${file}`, (err) => {
        if (err) throw err;
      });
    });
```